alerts/tibber.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Tibber is blocking old and dev version of Home Assistant"
```

Comment:
It's not so much blocking as it's migrating to a new endpoint? Is it public why the dev version also doesn't work ?

```suggestion
title: "Tibber is blocking old and dev version of Home Assistant"
```

Reply:
Home Assistant has not started to use the new endpoint yet, so the old endpoint is blocked for most user agents.
I do not know why the dev version does not work. Might be a mistake from Tibber.
(I did not know that it would happen so soon, so I have not had time to figure it out. )

Reply:
Got an answer from one of the backend developers. Dev version will be unblocked